lib/servers/WebsocketServer.js

```diff
@@ -40,7 +42,7 @@ module.exports = class WebsocketServer extends BaseServer {
         socket.isAlive = false;
         socket.ping(noop);
       });
-    }, this.server.webSocketHeartbeatInterval);
+    }, server.webSocketHeartbeatInterval || this.webSocketHeartbeatInterval);
```

Comment:
@alexander-akait had to make this change because we want to tune this value from test, can you think of any better approach?

Reply:
We need rewrite our test on e2e, because it is stupid test